platform/android/export/export.cpp

```diff
@@ -768,6 +768,81 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 	}
 
+	void _write_tmp_manifest(const Ref<EditorExportPreset> &p_preset, bool p_give_internet, bool p_debug) {
```

Comment:
Not all of it. For example, the `_get_permissions()` method would then have to be accessible outside of `export.cpp`. However, a lot of this can be moved out.

Reply:
This method is much smaller and easier to read after your updates, so it's fine to leave it in `export.cpp`.